packages/upscalerjs/src/constants.ts

```diff
@@ -2,4 +2,4 @@
 This is an autogenerated file. See scripts/pin-cdn.js for more information.
 *****/
 
-export const ROOT = 'https://unpkg.com/@upscalerjs/models@0.8.26';
+export const ROOT = 'https://unpkg.com/@upscalerjs/models@0.8.27';
```

Comment:
In my old local version, `@latest` is added by utils/buildUrl. Why go to hard-coded versions?